packages/backend/src/panels/YeomanUIPanel.ts

```diff
@@ -133,14 +133,14 @@ export class YeomanUIPanel extends AbstractWebviewPanel {
 
     let uri;
     try {
-      if (_.isEmpty(currentPath)) {
+      if (isEmpty(currentPath)) {
         throw new Error("Empty path");
       }
       uri = vscode.Uri.file(currentPath);
     } catch (e) {
```

Comment:
uri = vscode.Uri.file(currentPath);
if (!uri?.fsPath) {
    throw new Error("Empty path");
}

Reply:
I do not understand what how it helps ?
Please explain